src/token_metadatas_for_networks.ts

```diff
@@ -13,6 +13,7 @@ export interface TokenMetadataAndChainAddresses {
 }
 
 // Most token metadata taken from https://github.com/MetaMask/eth-contract-metadata/
+// And https://github.com/compound-finance/compound-protocol/blob/master/networks/kovan.json
 // tslint:disable:max-file-line-count
 export const TokenMetadatasForChains: TokenMetadataAndChainAddresses[] = [
     {
```

Comment:
Can't click on DAI address, but this one might be SAI for Kovan.

https://kovan.etherscan.io/address/0x4F96Fe3b7A6Cf9725f59d353F723c1bDb64CA6Aa 

https://github.com/makerdao/developerguides/tree/master/remittance#live-dai-token-smart-contract-deployments

Reply:
I get a `VM execution error` when using this one.